### PR TITLE
ProtelisDoc plugin in protelis-lang for generation of docs

### PR DESCRIPTION
**Pull request by metaphori**

### CODE REVIEW

**Comment by DanySK** [comment]
Can you provide a snapshot preview of protelis-lang?

**Comment by DanySK** [comment]
@metaphori last changes made the merge impossible: can you update it?

File: protelis/protelis-lang/build.gradle [hunk 1]
```diff
@@ -0,0 +1,22 @@
+plugins {
```
Comment (DanySK) [inline protelis/protelis-lang/build.gradle]: Nope.
The build system is now in kts format, that is the point where the plugin should get applied. Do not regenerate dead files, please.
This is a no-go.

**Comment by jakebeal** [comment]
What's holding this one up at this point?

**Comment by metaphori** [comment]
A snapshot

image

**Comment by DanySK** [comment]
@metaphori should I close this and refer to #223 instead?

**Comment by metaphori** [comment]
image

**Comment by metaphori** [comment]
> @metaphori should I close this and refer to #223 instead?

Dunno, but I notice that #223 may not be the last PR that (partially) completes this issue

**Pull request closed without merging**